### PR TITLE
Checkout Step One page tests

### PR DESCRIPTION
**Pull request by LiviuAC**

### CODE REVIEW

File: test/pageobjects/CheckoutPage.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                await this.btnContinue.click()
+                break
+            case "lettersZipCodeField":
+                await this.firstNameText.setValue("testFirstName")
+                await this.lastNameText.setValue("testLastName")
+                await this.zipCodeText.setValue("testZipCode")
+                await this.btnContinue.click()
+                break
+        }
+    }
```
Comment (axboory) [inline test/pageobjects/CheckoutPage.js]: It's not necessary to use such a big switch case when you can just simplify it like this `async fillForm(firstName, lastName, zipCodeText) {
                await this.firstNameText.setValue(firstName)
                await this.lastNameText.setValue(lastName)
                await this.zipCodeText.setValue(zipCodeText)
                await this.btnContinue.click()
}
`
and then just puting what parameters you need when calling the method.

**Review by axboory — CHANGES_REQUESTED** [review]
what is the .DS_Store file? and didn't you already merged the cartPage tests? Please ALWAYS check that everything is OK before push and when creating a pull request.

File: test/pageobjects/CheckoutPage.js (hunk 1 above)
Comment (LiviuAC) [reply test/pageobjects/CheckoutPage.js]: the method has been modified as suggested

**Comment by LiviuAC** [comment]
Regarding the .DS_Store file it is a file which is displayed only on the git repo. it is a file related macOS and couldn't be found locally. 

Note that, before push, upon checking the status, it was displayed on my terminal with red and it wasn't added.

After further investigation, it was found in several places with "git status" and added in the .gitignore fire. We will try to remove it after the next merge.

Regarding the cartPage tests, the merge was done before correcting the small typo. It has been corrected and it won't happen again.

**Review by axboory — APPROVED** [review]

**Pull request merged**